src/ConnectionManager.ts

```diff
@@ -205,7 +205,7 @@ export class ConnectionManager extends EventEmitter {
 
     public getConnectionsForGitLabRepo(pathWithNamespace: string): GitLabRepoConnection[] {
         pathWithNamespace = pathWithNamespace.toLowerCase();
-        return this.connections.filter((c) => (c instanceof GitLabRepoConnection && c.path === pathWithNamespace)) as GitLabRepoConnection[];
+        return this.connections.filter((c) => (c instanceof GitLabRepoConnection && c.path.toLowerCase() === pathWithNamespace)) as GitLabRepoConnection[];
```

Comment:
I'd make `c.path` itself lowercase in `GitLabRepoConnection`, and check all usages of it. We shouldn't be having mixed-case state keys as a rule.

Reply:
That would cause a force-lowercased path to be sent to `GitLabClient.issues.create`.  Could that be a problem?

Reply:
> That would cause a force-lowercased path to be sent to `GitLabClient.issues.create`. Could that be a problem?

Should be fine :+1:

Reply:
Tested & confirmed that `GitLabClient.issues.create` is case insensitive after all :+1: